[debug] [2020-04-11T22:50:15.887Z] ----------------------------------------------------------------------
[debug] [2020-04-11T22:50:15.890Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\mjarvis\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-11T22:50:15.891Z] CLI Version:   8.0.2
[debug] [2020-04-11T22:50:15.891Z] Platform:      win32
[debug] [2020-04-11T22:50:15.891Z] Node Version:  v10.16.3
[debug] [2020-04-11T22:50:15.892Z] Time:          Sat Apr 11 2020 16:50:15 GMT-0600 (Mountain Daylight Time)
[debug] [2020-04-11T22:50:15.892Z] ----------------------------------------------------------------------
[debug] [2020-04-11T22:50:15.892Z] 
[debug] [2020-04-11T22:50:15.907Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-11T22:50:15.952Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\mjarvis\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-11T22:52:05.587Z] ----------------------------------------------------------------------
[debug] [2020-04-11T22:52:05.590Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\mjarvis\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init --debug
[debug] [2020-04-11T22:52:05.591Z] CLI Version:   8.0.2
[debug] [2020-04-11T22:52:05.591Z] Platform:      win32
[debug] [2020-04-11T22:52:05.591Z] Node Version:  v10.16.3
[debug] [2020-04-11T22:52:05.592Z] Time:          Sat Apr 11 2020 16:52:05 GMT-0600 (Mountain Daylight Time)
[debug] [2020-04-11T22:52:05.592Z] ----------------------------------------------------------------------
[debug] [2020-04-11T22:52:05.592Z] 
[debug] [2020-04-11T22:52:05.602Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-11T22:52:05.632Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\mjarvis\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-11T22:52:28.020Z] ----------------------------------------------------------------------
[debug] [2020-04-11T22:52:28.024Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\mjarvis\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init --debug
[debug] [2020-04-11T22:52:28.024Z] CLI Version:   8.0.2
[debug] [2020-04-11T22:52:28.024Z] Platform:      win32
[debug] [2020-04-11T22:52:28.024Z] Node Version:  v10.16.3
[debug] [2020-04-11T22:52:28.025Z] Time:          Sat Apr 11 2020 16:52:28 GMT-0600 (Mountain Daylight Time)
[debug] [2020-04-11T22:52:28.025Z] ----------------------------------------------------------------------
[debug] [2020-04-11T22:52:28.026Z] 
[debug] [2020-04-11T22:52:28.035Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-11T22:52:28.065Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\mjarvis\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-11T23:04:14.429Z] ----------------------------------------------------------------------
[debug] [2020-04-11T23:04:14.432Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\mjarvis\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-11T23:04:14.432Z] CLI Version:   8.0.2
[debug] [2020-04-11T23:04:14.433Z] Platform:      win32
[debug] [2020-04-11T23:04:14.433Z] Node Version:  v10.16.3
[debug] [2020-04-11T23:04:14.434Z] Time:          Sat Apr 11 2020 17:04:14 GMT-0600 (Mountain Daylight Time)
[debug] [2020-04-11T23:04:14.434Z] ----------------------------------------------------------------------
[debug] [2020-04-11T23:04:14.434Z] 
[debug] [2020-04-11T23:04:14.443Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-11T23:04:14.495Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\mjarvis\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
